src/taco-dependency-installer/installers/androidPackagesInstaller.ts

```diff
@@ -77,26 +82,25 @@ class AndroidPackagesInstaller extends InstallerBase {
         }
     }
 
-    private getAvailableAndroidPackages(): Q.Promise<any> {
-        var deferred: Q.Deferred<any> = Q.defer();
+    private executeAndroidCommand(name: string, args: string[]): IExecuteCommandResult {
```

Comment:
First comment: I can't think of any other Android command we would need other than list and update. But even if we did add new methods, and they don't involve a prompt, I still think it's good to have the low-level details of how to bypass a prompt hidden in the execute command. Can we instead have a parameter `expectPrompt: boolean = false` to trigger this logic, and another `acceptPrompt: boolean = true`? This way, the callers are still in control, but we don't have to duplicate the prompt logic in every caller

Second command: I like your new name and I also agree to leave it mandatory